neutron/templates/configmaps/openvswitch-ensure-configured.sh.yaml.j2

```diff
@@ -16,7 +16,7 @@ data:
 
     if [[ ! $(ovs-vsctl list-ports $bridge) =~ $(echo "\<$port\>") ]]; then
         changed=changed
-        #ovs-vsctl --no-wait add-port $bridge $port
+        ovs-vsctl --no-wait add-port $bridge $port
```

Comment:
We would need a separate setting (false by default) which will check if deployer wants this to happen.